test/language/computed-property-names/computed-property-names-prototype.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    computed property names can be "prototype"
+---*/
+class C {
+  static ['prototype']() {
```

Comment:
Some mjsunit tests are not (or no longer) correct. https://github.com/anba/es6draft/blob/master/src/test/resources/mjsunit.list contains more infos (but I cannot guarantee that list is up-to-date because it requires manual updates :unamused: ).